app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -1,11 +1,17 @@
 module ProviderInterface
   class ProviderUsersController < ProviderInterfaceController
-    before_action :requires_provider_add_provider_users_feature_flag, only: %i[new create]
+    before_action :requires_provider_add_provider_users_feature_flag
+    before_action :redirect_unless_permitted_to_manage_users
 
     def index
       @provider_users = ProviderUser.visible_to(current_provider_user)
     end
 
+    def show
+      @provider_user = ProviderUser.find_by(id: params[:id])
```

Comment:
This allows someone with manage_users access to a provider to see all provider users in the system by changing the ID in the URL. 

It also allows them to change their provider organisations (but only to the ones the current user has access to).

Reply:
Thanks - have scoped this to the `current_provider_user` now